Utilities/Thread.cpp

```diff
@@ -1556,7 +1556,7 @@ bool handle_access_violation(u32 addr, bool is_writing, x64_context* context) no
 			if (!g_tls_access_violation_recovered)
 			{
 				vm_log.notice("\n%s", dump_useful_thread_info());
-				vm_log.error("Access violation %s location 0x%x (%s) [type=u%u]", is_writing ? "writing" : "reading", addr, (is_writing && vm::check_addr(addr)) ? "read-only memory" : "unmapped memory", d_size * 8);
+				vm_log.always("SPU Access violation %s location 0x%x (%s) [type=u%u]", is_writing ? "writing" : "reading", addr, (is_writing && vm::check_addr(addr)) ? "read-only memory" : "unmapped memory", d_size * 8);
```

Comment:
Why?

Reply:
Because its fatal.

Reply:
always does not convey an error. it's just informational.

Reply:
Well "always" shouldn't be used at all, use "fatal" as highest severity log.

Reply:
It needs a hack like Emu.Resume() after it, ayways its out of the scope of this pr.